src/pathwayvis/services/ws.ts

```diff
@@ -90,10 +88,9 @@ export class WSService {
     }
     const requestId = this._generateID();
 
-    _.assign(data, {
+    Object.assign(data, {
```

Comment:
I know I'm nitpicky but using `Object.assign` like this is bad practice (the lodash code wasn't any better). Here you're modifying the `data` input parameter. `Object.assign` merges all the parameters to the first one, so the clean way of using it is
```
const payload = Object.assing({}, data, {'request-id': requestId});
```
you can already transform this into a json string:
```
const payload = JSON. stringify(Object.assing({}, data, {'request-id': requestId}));

// ...
// and pass payload as the data parameter:
  data: payload,
}
```